deployments/examples/ocis_ldap/docker-compose.yml

```diff
@@ -76,7 +76,8 @@ services:
       IDP_LDAP_LOGIN_ATTRIBUTE: "uid"
       IDP_LDAP_UUID_ATTRIBUTE: "ownclouduuid"
       IDP_LDAP_UUID_ATTRIBUTE_TYPE: binary
-      GRAPH_LDAP_SERVER_WRITE_ENABLED: "false" # assuming the external ldap is readonly
+      GRAPH_LDAP_SERVER_WRITE_ENABLED: "true" # assuming the external ldap is readonly
```

Comment:
We either need to change the value back to `false` or adapt the comment here.

Sorry, I overlooked this during the first round of review. :man_facepalming:

Reply:
I think changing the comment is the better move, as in this setup. the LDAP will always be writable anyway.